x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -719,6 +789,7 @@ export const getGroupedESQuery = (
               },
             },
           },
+          ...getContextAggregation(params),
```

Comment:
The retrieval code and the tests suggest that the additional agg should be nested inside of `filtered_results`.

Reply:
That's right. It should have been inside `filtered_results`. I have fixed it.